verify-cborld-presentation/README.md

```diff
@@ -0,0 +1,22 @@
+# Verify VC presenation via QR code
+
+### Description
+
+The app will allow you to scan a CBOR VC presentation in a QR code, decode and verify it using MATTR services.
+
+## Set Up
+
+1.  A VC presentation in a QR code: the presentation should be in CBOR data format, then compressed using gzip, then
```

Comment:
Is it gzip or deflate?

Reply:
Good question, deflate seems to wrap gzip but ill update code to use gzip instead of deflate